compliance_checker/cf/cf.py

```diff
@@ -1893,14 +1894,12 @@ def check_calendar(self, ds):
                 continue
             reasoning = None
             has_calendar = hasattr(v, 'calendar')
-            if not has_calendar:
-                reasoning = ['Variable %s should have a calendar attribute' % k]
             result = Result(BaseCheck.LOW,
                             has_calendar,
                             '§4.4.1 Time and calendar',
                             reasoning)
```

Comment:
We don't need a check that checks if something has a `calendar` attribute, only that the value of `calendar` is in a list of acceptable values.